.github/workflows/release.yml

```diff
@@ -185,7 +185,7 @@ jobs:
           JIRA_DESCRIPTION: ${{ env.jira_description }}
         run: |
           COMMIT_DETAILS=$(echo "${COMMIT_DETAILS}" | sed 's/\\:/:/g')
-          RELEASE_NOTES="✅ New *${{ github.event.repository.name }}* Release [${{ needs.version-tag.outputs.version }}](${RELEASE_URL}) $(date)\n\n${RELEASE_NOTES}\n\nCommit Details:\n${COMMIT_DETAILS}\n\nPR Description:\n${PR_DESCRIPTION}\n\nTicket Details:\n${JIRA_SUMMARY}\n${JIRA_DESCRIPTION}"
+          RELEASE_NOTES="✅ New *${{ github.event.repository.name }}* Release [${{ needs.version-tag.outputs.version }}](${RELEASE_URL}) $(date)\n${RELEASE_NOTES}\nCommit Details:\n${COMMIT_DETAILS}\nPR Description:\n${PR_DESCRIPTION}\nTicket Details:\n${JIRA_SUMMARY}\n${JIRA_DESCRIPTION}"
```

Comment:
Also, it's important to test the output of the `RELEASE_NOTES` after these modifications to make sure it still meets the desired format and requirements.